apps/src/p5lab/poetry/PoemSelector.jsx

```diff
@@ -1,21 +1,98 @@
-import React from 'react';
+import React, {useState} from 'react';
```

Comment:
@madelynkasula React state hooks! It's so clean!! I'm so excited to start using hooks everywhere!!

Reply:
yayyyy 🎉 i think this might be the first hooks usage in our codebase!